[2022-12-13T01:24:34.667] [ERROR] Error - Error en el servidor: listen EADDRINUSE: address already in use :::8080
[2022-12-13T01:25:15.204] [ERROR] Error - No existe el usuario con el mail indicado.
[2022-12-16T00:41:52.674] [ERROR] Error - Error: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
    at Sequence._packetToError (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
    --------------------
    at Protocol._enqueue (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/Connection.js:116:18)
    at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/dialects/mysql/index.js:66:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/dialects/mysql/index.js:61:12)
    at create (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/client.js:249:39) {
  code: 'ER_NOT_SUPPORTED_AUTH_MODE',
  errno: 1251,
  sqlMessage: 'Client does not support authentication protocol requested by server; consider upgrading MySQL client',
  sqlState: '08004',
  fatal: true
}
[2022-12-16T00:42:05.124] [ERROR] Error - Error: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
    at Sequence._packetToError (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
    --------------------
    at Protocol._enqueue (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mysql/lib/Connection.js:116:18)
    at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/dialects/mysql/index.js:66:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/dialects/mysql/index.js:61:12)
    at create (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/client.js:249:39) {
  code: 'ER_NOT_SUPPORTED_AUTH_MODE',
  errno: 1251,
  sqlMessage: 'Client does not support authentication protocol requested by server; consider upgrading MySQL client',
  sqlState: '08004',
  fatal: true
}
[2022-12-17T13:20:27.210] [ERROR] Error - Error: Undefined binding(s) detected when compiling SELECT. Undefined column(s): [id] query: select * from `products` where `id` = ?
    at QueryCompiler_MySQL.toSQL (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/query/querycompiler.js:110:13)
    at Builder.toSQL (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/query/querybuilder.js:83:44)
    at ensureConnectionCallback (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/execution/internal/ensure-connection-callback.js:4:30)
    at Runner.ensureConnection (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/execution/runner.js:300:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Runner.run (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/execution/runner.js:30:19)
    at async getElementById (file:///home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/src/utils/knex/getElementById.js:7:21)
    at async ProductsDAOMySQL.readProductById (file:///home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/src/persistency/DAO/products/mySQL.js:54:23)
    at async ProductService.getProduct (file:///home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/src/services/graphql/products.service.js:31:23)
    at async graphqlMiddleware (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/express-graphql/index.js:125:26)
[2022-12-17T13:27:56.920] [ERROR] Error - Error: Undefined binding(s) detected when compiling SELECT. Undefined column(s): [id] query: select * from `products` where `id` = ?
    at QueryCompiler_MySQL.toSQL (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/query/querycompiler.js:110:13)
    at Builder.toSQL (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/query/querybuilder.js:83:44)
    at ensureConnectionCallback (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/execution/internal/ensure-connection-callback.js:4:30)
    at Runner.ensureConnection (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/execution/runner.js:300:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Runner.run (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/knex/lib/execution/runner.js:30:19)
    at async getElementById (file:///home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/src/utils/knex/getElementById.js:7:21)
    at async ProductsDAOMySQL.readProductById (file:///home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/src/persistency/DAO/products/mySQL.js:54:23)
    at async ProductService.getProduct (file:///home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/src/services/graphql/products.service.js:31:23)
    at async graphqlMiddleware (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/express-graphql/index.js:125:26)
[2022-12-17T16:10:18.333] [ERROR] Error - Error: Messages validation failed: fyh: Cast to date failed for value "17/12/2022, 16:10:18" (type string) at path "fyh"
    at ValidationError.inspect (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/error/validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/layouts.js:65:52)
    at app (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/appenders/file.js:98:23)
    at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/log4js.js:43:5
    at Array.forEach (<anonymous>)
    at sendLogEventToAppender (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/log4js.js:42:21) {
  errors: {
    fyh: CastError: Cast to date failed for value "17/12/2022, 16:10:18" (type string) at path "fyh"
        at SchemaDate.cast (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schema/date.js:354:11)
        at SchemaType.applySetters (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schematype.js:1201:12)
        at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1421:22)
        at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1146:16)
        at model.Document (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:166:12)
        at model.Model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:121:12)
        at new model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:5039:15)
        at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3187:22
        at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3222:7
        at Array.forEach (<anonymous>) {
      stringValue: '"17/12/2022, 16:10:18"',
      messageFormat: undefined,
      kind: 'date',
      value: '17/12/2022, 16:10:18',
      path: 'fyh',
      reason: AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
      
        assert.ok(false)
      
          at castDate (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/cast/date.js:40:10)
          at SchemaDate.cast (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schema/date.js:352:12)
          at SchemaType.applySetters (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schematype.js:1201:12)
          at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1421:22)
          at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1146:16)
          at model.Document (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:166:12)
          at model.Model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:121:12)
          at new model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:5039:15)
          at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3187:22
          at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3222:7 {
        generatedMessage: true,
        code: 'ERR_ASSERTION',
        actual: false,
        expected: true,
        operator: '=='
      },
      valueType: 'string'
    }
  },
  _message: 'Messages validation failed'
}
[2022-12-17T16:17:36.654] [ERROR] Error - Error: Messages validation failed: fyh: Cast to date failed for value "17/12/2022, 16:17:36" (type string) at path "fyh"
    at ValidationError.inspect (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/error/validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/layouts.js:65:52)
    at app (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/appenders/file.js:98:23)
    at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/log4js.js:43:5
    at Array.forEach (<anonymous>)
    at sendLogEventToAppender (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/log4js.js:42:21) {
  errors: {
    fyh: CastError: Cast to date failed for value "17/12/2022, 16:17:36" (type string) at path "fyh"
        at SchemaDate.cast (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schema/date.js:354:11)
        at SchemaType.applySetters (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schematype.js:1201:12)
        at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1421:22)
        at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1146:16)
        at model.Document (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:166:12)
        at model.Model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:121:12)
        at new model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:5039:15)
        at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3187:22
        at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3222:7
        at Array.forEach (<anonymous>) {
      stringValue: '"17/12/2022, 16:17:36"',
      messageFormat: undefined,
      kind: 'date',
      value: '17/12/2022, 16:17:36',
      path: 'fyh',
      reason: AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
      
        assert.ok(false)
      
          at castDate (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/cast/date.js:40:10)
          at SchemaDate.cast (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schema/date.js:352:12)
          at SchemaType.applySetters (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schematype.js:1201:12)
          at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1421:22)
          at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1146:16)
          at model.Document (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:166:12)
          at model.Model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:121:12)
          at new model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:5039:15)
          at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3187:22
          at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3222:7 {
        generatedMessage: true,
        code: 'ERR_ASSERTION',
        actual: false,
        expected: true,
        operator: '=='
      },
      valueType: 'string'
    }
  },
  _message: 'Messages validation failed'
}
[2022-12-17T16:20:51.886] [ERROR] Error - Error: Messages validation failed: fyh: Cast to date failed for value "17/12/2022, 16:20:51" (type string) at path "fyh"
    at ValidationError.inspect (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/error/validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/layouts.js:65:52)
    at app (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/appenders/file.js:98:23)
    at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/log4js.js:43:5
    at Array.forEach (<anonymous>)
    at sendLogEventToAppender (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/log4js/lib/log4js.js:42:21) {
  errors: {
    fyh: CastError: Cast to date failed for value "17/12/2022, 16:20:51" (type string) at path "fyh"
        at SchemaDate.cast (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schema/date.js:354:11)
        at SchemaType.applySetters (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schematype.js:1201:12)
        at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1421:22)
        at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1146:16)
        at model.Document (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:166:12)
        at model.Model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:121:12)
        at new model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:5039:15)
        at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3187:22
        at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3222:7
        at Array.forEach (<anonymous>) {
      stringValue: '"17/12/2022, 16:20:51"',
      messageFormat: undefined,
      kind: 'date',
      value: '17/12/2022, 16:20:51',
      path: 'fyh',
      reason: AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
      
        assert.ok(false)
      
          at castDate (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/cast/date.js:40:10)
          at SchemaDate.cast (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schema/date.js:352:12)
          at SchemaType.applySetters (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/schematype.js:1201:12)
          at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1421:22)
          at model.$set (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:1146:16)
          at model.Document (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/document.js:166:12)
          at model.Model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:121:12)
          at new model (/home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:5039:15)
          at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3187:22
          at /home/alexisvp97/Escritorio/CursoBackend/Desafios/Desafio_19/node_modules/mongoose/lib/model.js:3222:7 {
        generatedMessage: true,
        code: 'ERR_ASSERTION',
        actual: false,
        expected: true,
        operator: '=='
      },
      valueType: 'string'
    }
  },
  _message: 'Messages validation failed'
}
